test/Ajax.js

```diff
@@ -195,6 +198,16 @@ describe('Ajax', function() {
 			this.requests[0].error();
 		});
 
+		it('should fail when attempting to fetch resource from cross domain', function(done) {			
+			this.xhr.restore();
```

Comment:
FakeXMLHttpRequest cannot support CORS activities and therefore cannot trigger or imitate CORS exceptions. Here the test restores the original XMLHttpRequest object that will be used throughout this test scenario.